src/browser-frontend/views/browser/chrome/tabbar/tab.jsx

```diff
@@ -45,7 +48,9 @@ export default class Tab extends PureComponent {
     return (
       <a
         tabIndex={0}
-        styleName={`tab ${this.props.selected ? 'selected' : ''} ${this.props.tabState}`}
+        styleName={`tab \
+          ${this.props.selected ? 'selected' : ''} ${this.props.tabState} \
+          ${this.props.loadState === DomainPageMetaModel.LOAD_STATES.LOADED ? this.props.loadState : ''}`}
```

Comment:
These checks are getting a bit gnarly and out of hand. Some helper function would be awesomesauce for these kinds of things. Something like `classIfEq(this.props.loadState, DomainPageMetaModel.LOAD_STATES.LOADED)`, and maybe even a `classIfNotEq`, where the signatures would be something like `classIfEq/onlyIfNotEq(src, ...dst)`